ACHNBrowserUI/ACHNBrowserUI/environements/Items.swift

```diff
@@ -12,19 +12,21 @@ import Combine
 class Items: ObservableObject {
     static let shared = Items()
     
-    @Published var categories: [Categories: [Item]] = [:]
+    @Published var categories: [Category: [Item]] = [:]
     
-    private var cancellable: [AnyCancellable] = []
+    private var cancellable = Set<AnyCancellable>()
```

Comment:
I can revert that if you don't like this approach, but it's more conventional to see `cancellables` with `Set<AnyCancellable>`.
Idem for the `.store(in: &cancellable)` instead of `.append(...)` in front of the Publisher.
Tell me if it's OK to go this way.

Reply:
It's fine. Ideally we don't want to store them anyway, it's just to keep them referenced so the query to load categories keep running.